src/lib/features/tag-type/tag-types.e2e.test.ts

```diff
@@ -139,6 +200,36 @@ test('Validation of tag-types returns 200 for valid tag-types', async () => {
         });
 });
 
+test('Validation of tag-types with valid color returns 200', async () => {
```

Comment:
"Successful validation of tag type color". 200 is impl detail not relevant for the test name